cfpb/cfpb-chart-builder
# Consolidate browsersync in watch task

## Changes

- Moves browserSync task and config into the `gulp watch` task, since it doesn't make sense to use it on its own.
- Removes browser-sync cruft from style and script tasks and use `.on( 'change' … )` event on the watch task instead.
- Convert task functions to arrow functions.
- Update `browser-sync`, and `compression` to latest.

## Testing

- Run `./setup.sh`
- Run `gulp watch`
- Edit some CSS and JS and see that build runs in Terminal and page in browser reloads with change (adding an outline for CSS or an alert in JS is a good way to check).
